api/v1/openapi.yaml

```diff
@@ -1348,6 +1377,14 @@ definitions:
         type: boolean
       datapath-configuration:
         "$ref": "#/definitions/EndpointDatapathConfiguration"
+  EndpointDeleteRequest:
```

Comment:
This is a bit confusing as the commit is adding a new API to delete a batch of endpoints. There is already an API to delete endpoints by ID. Can we clearly differentiate between the two?

Reply:
Good point. I've renamed it to `EndpointBatchDeleteRequest` (and the parameter type from `endpoint-delete-request` to `endpoint-batch-delete-request`).